Example/ImageKit/UploadAuthService.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  UploadAuthService.swift
+//  ImageKit_Example
+//
+//  Created by Animesh Verma on 25/09/23.
+//  Copyright © 2023 CocoaPods. All rights reserved.
+//
+
+import Foundation
+
+class UploadAuthService {
+    static let dispatchGroup = DispatchGroup()
+    
+    static func getUploadToken(payload: [String : String]) -> [String : String]? {
+        let urlSession = URLSession(configuration: URLSessionConfiguration.default, delegate: URLSession.shared.delegate, delegateQueue: URLSession.shared.delegateQueue)
+        var request = URLRequest(url: URL(string: "AUTH_SERVER_HOST")!)
+        request.httpMethod = "POST"
+        request.setValue("application/json", forHTTPHeaderField: "Content-Type")
+        var tokenResponse: [String : String]? = nil
+        guard let body = try? JSONSerialization.data(withJSONObject: ["uploadPayload": payload, "expire": 60, "publicKey": "IK_PUBLIC_KEY"] as [String : Any]) else {
```

Comment:
`IK_PUBLIC_KEY` should come from a constants file

Reply:
Done

Reply:
This one value is still left, should come from constants

Reply:
Done